pyproject.toml

```
@@ -11,31 +11,36 @@ dynamic = ["version"]
 description = "A NOMAD plugin for analysis of FAIR data."
 readme = "README.md"
 authors = [
-    { name = "Sarthak Kapoor", email = "sarthak.kapoor@physik.hu-berlin.de" }
+    { name = "Sarthak Kapoor", email = "sarthakkapoor@hotmail.com" },
 ]
-license = { text = "Apache-2.0" }
+maintainers = [
+    { name = "Sarthak Kapoor", email = "sarthakkapoor@hotmail.com" },
```

We will add all the team also in the others

```suggestion
    { name = "Sarthak Kapoor", email = "sarthakkapoor@hotmail.com" },
    { name = "Hampus Näsström", email = 'hampus.naesstroem@physik.hu-berlin.de' },
    { name = "Andrea Albino", email = 'andrea.albino@physik.hu-berlin.de' },
    { name = "Sebastian Brückner", email = 'sebastian.brueckner@ikz-berlin.de' },
```